.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,90 @@
+name: "\U0001F41B Bug Report"
+description: Report a bug encountered while using napari
+labels:
+  - "bug"
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to report this issue! 🙏🏼
+
+        Please fill out the sections below to help us reproduce the problem.
+
+        If you've found a problem with content on the napari documentation site (napari.org) or with the rendering of the content,
+        please let us know [here](https://github.com/napari/docs/issues)
+
+  - type: textarea
+    id: bug-report
+    attributes:
+      label: "\U0001F41B Bug Report"
+      description: "Please provide a clear and concise description of the bug."
+      placeholder: "What went wrong? What did you expect to happen?"
+    validations:
+      required: true
+
+  - type: textarea
+    id: steps-to-reproduce
+    attributes:
+      label: "\U0001F4A1 Steps to Reproduce"
+      description: "Please provide a minimal code snippet or list of steps to reproduce the bug."
+      placeholder: |
+        1. Go to '...'
+        2. Click on '....'
+        3. Scroll down to '....'
+        4. See error
+    validations:
+      required: true
+
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: "\U0001F4A1 Expected Behavior"
+      description: "Please provide a clear and concise description of what you expected to happen."
+      placeholder: "What did you expect to happen?"
+
+  - type: textarea
+    id: environment
+    attributes:
+      label: "\U0001F30E Environment"
+      description: |
+        Please provide the following information about your environment. You could just paste the output of `napari --info` here or copy  the information from the "napari infor" dialog in the napari Help menu.
+        
+        Otherwise, please provide information about your Operating system, Python version, napari version, Qt backend and version, Qt platform, and any other relevant information about your environment:
```

Comment:
```suggestion
        Otherwise, please provide information regarding your operating system (OS), Python version, napari version, Qt backend and version, Qt platform, method of installation, and any other relevant information related to your environment.
```